lib/awful/widget/layoutbox.lua

```diff
@@ -47,9 +46,9 @@ function layoutbox.new(screen)
 
     -- Do we already have the update callbacks registered?
     if boxes == nil then
-        boxes = setmetatable({}, { __mode = "v" })
-        tag.attached_connect_signal(nil, "property::selected", update_from_tag)
-        tag.attached_connect_signal(nil, "property::layout", update_from_tag)
+        boxes = setmetatable({}, { __mode = "kv" })
```

Comment:
Sigh, I did the same change in #832 and thus will have to do another rebase if this is merged....